src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -189,14 +205,15 @@ def register_content_settings_argument(scope, settings_class):
     register_cli_argument('storage blob {}'.format(item), 'validate_content', action='store_true')
 
 for item in ['update', 'upload']:
-    register_content_settings_argument('storage blob {}'.format(item), BlobContentSettings)
+    register_content_settings_argument('storage blob {}'.format(item), BlobContentSettings, item == 'update')
```

Comment:
It will soon be more than one line (part 2) so I'll leave it for now.

---
In reply to: [77392257](https://github.com/Azure/azure-cli/pull/804#discussion_r77392257) [](ancestors = 77392257)